cognitedata/reveal
# fix: error occuring when preloading image with no annotations

#### Type of change

![Bug](https://img.shields.io/badge/Type-Bug-red) 

#### Jira ticket :blue_book:


https://cognitedata.atlassian.net/browse/

## Description :pencil:


An error used to occur when preloading 360 image annotations for images with no annotations, because `.add` does not handle zero-argument inputs.

## How has this been tested? :mag:



- Open any 360 image (e.g. `helideck-site-2-jpeg`, with https://localhost:3000/?project=3d-test&env=greenfield-3d&modelId=4923307033982497&revisionId=3100284276970593)
- Observe that hovering the pointer over 360 images without annotations does not trigger errors in the log

## Test instructions :information_source:




## Checklist :ballot_box_with_check:
